tools/source_updater/README.md

```diff
@@ -0,0 +1,39 @@
+# Source update
```

Comment:
idea (doesn't need to be done in this PR): Is it worth adding these new tools to airbyte-ci?

Reply:
If we agree that the end goal is to have this as part of the connector builder, I would think this means it should be moved to `airbyte_cdk.conenctor_builder`. As a transition step, it can still be a script in this package that we can eventually expose through HTTP. I'll trigger the discussion tomorrow in the grooming session